# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/base/workbone/qd-fixes.patch
# ROCK Linux is Copyright (C) 1998 - 2006 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./hardware.c
+++ ./hardware.c	Mon Oct 18 16:16:27 1999
@@ -14,6 +14,8 @@
 #include <stdlib.h>
 #include <unistd.h>
 #include <sys/types.h>
+#include <sys/types.h>
+#include <ustat.h>
 #include <sys/ioctl.h>
 #include <fcntl.h>
 #include <sys/param.h>
@@ -25,9 +25,6 @@
 #    else
 #     include <sundev/srreg.h>
 #    endif /* linux */
-#ifdef __GLIBC__
-#include <sys/ustat.h>
-#endif
 #include "struct.h"
 #include "workbone.h"
